### PR TITLE
Added VAT validation for Switzerland.

### PR DESCRIPTION
**Pull request by lukapopovic**
Hey guys, I am using your small library and find it really simple and useful. I ran into problem with swiss VAT though. I hope you don't mind the pull request. 

Regards,
Luka

### CODE REVIEW

File: patterns.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ var RO = "^RO[0-9]{2,10}$";
 var SE = "^SE[0-9]{10,10}01$";
 var SI = "^SI[0-9]{8,8}$";
 var SK = "^SK[0-9]{10,10}$";
+var CH = "^CHE[0-9]{9,9}$";
```
Comment (elliotblackburn) [inline patterns.js]: Just checking out the wikipedia article on non-EU VAT codes and I found the following about Swiss VAT codes:

> 6 digits (up to 31/12/2013). CHE 9 numeric digits plus TVA/MWST/IVA e.g. CHE-123.456.788 TVA[20] The last digit is a MOD11 checksum digit build with weighting pattern: 5,4,3,2,7,6,5,4[21]

The format you've supplied for the tests below, and the regex here, don't include the intermediate dots. Is excluding the dots common in Switzerland when entering a VAT code, or are they often seen with the dots? I also noticed the missing hyphen as well.

I'd be incline to prefer sticking to the Swiss standards of using the hyphen and dots but if this isn't actually used by anyone then the pragmatic approach would be as you've got here.

Do you have any thoughts? Being the one using Swiss VAT codes, I'm assuming you might have some better insight than me!

**Review by elliotblackburn — COMMENTED** [review]
Hi there @lukapopovic, thanks so much for your PR, I'm glad you're finding the module useful and I'm even happier that you've gone out of your way to send a PR to fix this issue.

I've got a few questions about Swiss VAT codes which I thought it'd be sensible to discuss a little before I test, merge, and publish this change.

I appreciate you may not be a VAT code expert (I'm certainly not), but if you're working with Swiss VAT codes then you might have greater knowledge than me!

**Comment by elliotblackburn** [comment]
Also I just noticed this is your first PR on github, congrats and thanks for making it this project! 🎉

File: patterns.js (hunk 1 above)
Comment (lukapopovic) [reply patterns.js]: Hi @BlueHatbRit, first of all thank you for the fast response! 👍

You are right, by the standards all of the variations are allowed:

`CHE112947556,
  CHE 112947556,
  CHE 112.947.556,
  CHE-112.947.556 IVA,
  CHE112.947.556TVA,
  CHE 112947556 MWST`

IMHO the first option is the usual one, but you never know from where will someone copy the VAT number and enter it in a different format. If we want to do it right all the options should be supported. That is why I prepared new regex which covers all the cases (at least all that I could think of :) ). I also added some more test cases. Take a look and let me know what you think!

**Comment by BlueHatbRit** [comment]
@lukapopovic I can see a new commit with some updates, they look great thanks for adding in those additional ones! Are you finished making your changes? If so I'll give them a test locally and then I'll merge and deploy!

**Comment by elliotblackburn** [comment]
Hey there, I've run the tests locally and it seems you've captured all the VAT code spec so I'm going to get this merged and deployed now. Thanks for your contribution and congrats on having your first pull request merged! :tada:

**Comment by elliotblackburn** [comment]
This has been released under 1.1.0 (https://github.com/BlueHatbRit/vatcode-validator/releases/tag/1.1.0) and can now be pulled from npm.

**Comment by lukapopovic** [comment]
Great Job @BlueHatbRit. Thank you so much! Cheers

**Pull request merged**